src/sentry/api/endpoints/organization_releases.py

```diff
@@ -126,6 +126,7 @@ def get(self, request, organization):
         with_health = request.GET.get("health") == "1"
         flatten = request.GET.get("flatten") == "1"
         sort = request.GET.get("sort") or "date"
+        chart_only = request.GET.get("chartOnly") == "1"
```

Comment:
I was wondering if we shouldn't have a 'timeseries' endpoint for releases. That would let us reduce the amount of transformation we need to do in the frontend as well.

A separate endpoint could let us use a larger per-page sizes, let us fetch only the properties we need from the database, and by-pass ORM hydration.

Reply:
:+1: So totally agree on all the benefits of the separate endpoint, realized the old implementation was still hitting snuba, so this new endpoint is taking a fraction of the time, for magnitudes more data 🥳 

But not sure if there was a specific transformation you had in mind with when you mentioned this?
> reduce the amount of transformation we need to do in the frontend